server/handlers.go

```diff
@@ -892,6 +895,16 @@ func (s *Server) getHandler(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Disposition", fmt.Sprintf("%s; filename=\"%s\"", disposition, filename))
 	w.Header().Set("Connection", "keep-alive")
 
+	if w.Header().Get("Range") == "" {
```

Comment:
> # github.com/dutchcoders/transfer.sh/server
> server/handlers.go:898:1: syntax error: unexpected <<, expecting }

When I try to: `go build -o transfersh main.go`

Was there anything else you wanted me to try on the server?